templates/job_request/detail.html

```diff
@@ -298,6 +298,14 @@ <h1 class="text-3xl tracking-tight break-words font-bold text-slate-900 sm:text-
         </dl>
       {% /card %}
 
+      {% if compare_url %}
+        {% #card title="Previous Job Request" %}
```

Comment:
For consistency with the page title...

```suggestion
        {% #card title="Previous job request" %}
```

Reply:
That said, I think I'd

* Not make the card conditional. (You may wish to add an `else`.)
* Make the card's purpose clearer

```suggestion
      {% #card title="Code comparison" %}
        {% if compare_url %}
          {% #list_group small=True %}
              {% #list_group_item href=compare_url %}
                Compare this job request's code to the previous job request's code
              {% /list_group_item %}
          {% /list_group %}
        {% endif %}
      {% /card %}
```